setup-references.mjs

```
@@ -2,3 +1,4 @@
+import { constants, accessSync, existsSync, mkdirSync } from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)